src/app/content/search/hooks.spec.ts

```diff
@@ -127,7 +127,7 @@ describe('hooks', () => {
         match: {} as any,
       });
 
-      expect(dispatch).toHaveBeenCalledWith(requestSearch('qwer'));
+      expect(dispatch).not.toHaveBeenCalledWith(requestSearch('qwer'));
```

Comment:
i don't think you were supposed to change this. `'searches for saved query on POP if it is differet from current query'` sounds right to me

Reply:
this test was failing, saw the response and changed it based on that, since we modified the thing that did a search on reload